IP get panics

```
$ packet-cli ip get -p $PACKET_PROJECT_ID
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x14505ec]

goroutine 1 [running]:
github.com/packethost/packet-cli/cmd.glob..func33(0x194e440, 0xc00000f1e0, 0x0, 0x2)
        /Users/gianarb/git/packet-cli/cmd/retrieve_ip.go:65 +0xa3c
github.com/spf13/cobra.(*Command).execute(0x194e440, 0xc00000f180, 0x2, 0x2, 0x194e440, 0xc00000f180)
        /Users/gianarb/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x29d
github.com/spf13/cobra.(*Command).ExecuteC(0x194fc00, 0xa, 0x0, 0x0)
        /Users/gianarb/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ea
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/gianarb/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
github.com/packethost/packet-cli/cmd.NewCli(0x4)
        /Users/gianarb/git/packet-cli/cmd/cli.go:34 +0x157
main.main()
        /Users/gianarb/git/packet-cli/main.go:32 +0x26
```

--- a/cmd/retrieve_ip.go
+++ b/cmd/retrieve_ip.go
@@ -62,7 +62,11 @@ packet ip get --reservation-id [reservation_UUID]
 			data := make([][]string, len(ips))
 
 			for i, ip := range ips {
-				data[i] = []string{ip.ID, ip.Address, ip.Facility.Code, strconv.FormatBool(ip.Public), ip.Created}
+				code := ""
+				if ip.Facility != nil {
+					code = ip.Facility.Code
+				}
+				data[i] = []string{ip.ID, ip.Address, code, strconv.FormatBool(ip.Public), ip.Created}
 			}
 			header := []string{"ID", "Address", "Facility", "Public", "Created"}
 

